services/device-connection/pom.xml

```diff
@@ -25,6 +25,16 @@
       <groupId>org.eclipse.hono</groupId>
       <artifactId>client-device-connection-infinispan</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>hono-service-base</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>org.eclipse.hono</groupId>
+          <artifactId>hono-client-adapter-amqp</artifactId>
```

Comment:
Wouldn't it make sense to remove the 'hono-service-base' dependency here and add the exclusion to the existing 'hono-service-base' in the parent 'hono-services' module?

Reply:
absolutely